### ISSUE TITLE
idea: improve printing dialog

### ISSUE THREAD

**Issue by edmundlaugasson** [aardappel/treesheets#47]
There is an idea to improve printing dialog. Currently there are small dots as grid but when e.g. printing to PDF (I assume also onto paper) then there are thin violet lines (seems to be same color as there were dots in TreeSheets). Possibly I would prefer to see same small dots then there could be option in grid replacement like: replace dots with lines. Also ability to change the grid color would be good to have. Currently e.g. Options->Pick custom color... did not do that.
Also there is page setup for printing where can be layout set and paper format but no header/footer and page numbering. I would also e.g. have thin line separating header or footer from the rest of page content like it is possible in LibreOffice Calc. So - possibly copying page setup from e.g. LibreOffice Calc would help easier to implement it.

**Comment by aardappel** [comment]
The printing dialog and the way printing works in general is built-in to wxWidgets, which I have very little ability to change, beyond rewriting it all from scratch.

**Comment by edmundlaugasson** [comment]
OK then - let it be if it is too complex. Optionally make possible change e.g. grid inside document before printing would work. I am just worrying about if there will be large documents then how to print them onto A4 papers if needed... But it seems that just zooming out the needed part and then printing will do the trick.

**Comment by aardappel** [comment]
Yes, sadly if a large document is meant for print, then organizing it in a page-friendly way is all that's possible right now.
